apps/pollbook/backend/src/store_multi_node.test.ts

```diff
@@ -157,8 +157,6 @@ test('offline undo with later real time check in', async () => {
   expect(localB.getCheckInCount()).toEqual(2);
 
   // PollbookB goes offline
-  peerB.setOnlineStatus(false);
```

Comment:
the triggering of online/offline in these tests never did anything and was more for readability in understanding the scenario I am imagining, I think the comments do enough to convey that so just removing